requirements.txt

```diff
@@ -1,7 +1,9 @@
 datajoint>=0.13
 element-interface>=0.4.0
+ipywidgets
 openpyxl
 plotly
 pyopenephys>=1.1.6
 seaborn
 intanrhsreader @ git+https://github.com/datajoint/intanrhsreader.git
```

Comment:
@JaerongA , should the `intanrhsreader` dependency be added to the requirements?

Reply:
I've just resolved the conflicts,, it should be `intanrhdreader`,  not `intanrhsreader`